apps/maestro/src/features/SendInterchainToken/SendInterchainToken.tsx

```diff
@@ -270,22 +270,25 @@ export const SendInterchainToken: FC<Props> = (props) => {
     });
   }, [props.balance.decimals, props.balance.tokenBalance, setValue]);
 
-  const isSameChainType = useMemo(() => {
-    return state.selectedToChain?.chain_type === props.sourceChain.chain_type;
+  const shouldUseSameAddress = useMemo(() => {
+    return (
+      state.selectedToChain?.chain_type === "evm" &&
+      props.sourceChain.chain_type === "evm"
```

Comment:
does that mean `shouldUseSameAddress` will return false for `ethereum` -> `flow`?

Reply:
can we use `supportWagmi` flag from our chain config?
![image](https://github.com/user-attachments/assets/65706107-31c8-4442-91f3-6158e360b524)